python/docs/source/_static/css/asciinema-player.css

```
@@ -0,0 +1,711 @@
+  opacity: 0;
+}
+div.ap-term pre.ap-term-text {
+  position: absolute;
+  inset: 0;
+  box-sizing: content-box;
+  overflow: hidden;
+  padding: 0;
+  margin: 0px;
+  display: block;
+  white-space: pre;
+  word-wrap: normal;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)